doc/articles/interop/wasm-javascript-1.md

```diff
@@ -57,70 +57,100 @@ The [philosophy of Uno](https://platform.uno/docs/articles/concepts/overview/phi
 
 Here is a list of helper methods used to facilitate the integration with the HTML DOM:
 
-* The method `base.SetStyle()` can be used to set a CSS Style on the HTML element. Example:
+* The method `element.SetCssStyle()` can be used to set a CSS Style on the HTML element. Example:
 
   ``` csharp
   // Setting only one CSS style
-  SetStyle("text-shadow", "2px 2px red");
-
+  this.SetCssStyle("text-shadow", "2px 2px red");
+  
   // Setting many CSS styles at once using C# tuples
-  SetStyle(("text-shadow", "2px 2px blue"), ("color", "var(--app-fg-color1)"));
+  this.SetCssStyle(("text-shadow", "2px 2px blue"), ("color", "var(--app-fg-color1)"));
   ```
 
-* The method `base.ResetStyle()` can be used to set CSS styles to their default values. Example:
+  
+
+* The `element.ClearCssStyle()` method extension can be used to set CSS styles to their default values. Example:
 
   ``` csharp
   // Reset text-shadow style to its default value
-  ResetStyle("text-shadow");
+  this.ClearCssStyle("text-shadow");
 
   // Reset both text-shadow and color to their default values
-  ResetStyle("text-shadow", "color");
+  this.ClearCssStyle("text-shadow", "color");
   ```
 
-* The `base.SetAttribute()` and `base.RemoteAttribute()` methods can be used to set HTML attributes on the element:
+* The `element.SetHtmlAttribute()` and `element.ClearHtmlAttribute()` method extensions can be used to set HTML attributes on the element:
 
   ``` csharp
   // Set the "href" attribute of an <a> element
-  SetAttribute("href", "#section2");
+  this.SetHtmlAttribute("href", "#section2");
 
   // Set many attributes at once
-  SetAttribute(("target", "_blank"), ("referrerpolicy", "no-referrer"));
+  this.SetHtmlAttribute(("target", "_blank"), ("referrerpolicy", "no-referrer"));
 
   // Remove attribute from DOM element
-  RemoveAttribute("href");
+  this.ClearHtmlAttribute("href");
+  ```
+
+* The `element.SetCssClass()` and `element.UnsetCssClass()` method extensions can be used to add or remove CSS classes to the HTML Element:
+
+  ``` csharp
+  // Add the class to element
+  this.SetCssClass("warning");
+  
+  // Remove class from element
+  this.UnsetCssClass("paused");
   ```
 
-* The method `base.SetHtmlContent()` can be used to set arbitrary HTML content as child of the control.
+* The `element.SetHtmlContent()` method extension can be used to set arbitrary HTML content as child of the control.
 
   ``` csharp
-  SetHtmlContent("<h2>Welcome to Uno Platform!</h2>");
+  this.SetHtmlContent("<h2>Welcome to Uno Platform!</h2>");
   ```
 
-  > Note: should not be used when children "managed" controls are present: doing so can result in inconsistent runtime errors because of desynchronized visual tree.
+  > **IMPORTANT**: should not be used when children "managed" controls are present: doing so can result in inconsistent runtime errors because of desynchronized visual tree.
```

Comment:
```suggestion
  > **IMPORTANT**: This method should not be used when children "managed" controls are present: doing so can result in inconsistent runtime errors because of desynchronized visual tree.
```